docs/examples/collect.py

```diff
@@ -166,6 +169,9 @@ def test_download_merge(self):
 parser.add_argument('--zoom', type=int, default=12,
                     help='Map zoom level given as integer. Defaults to 12.')
 
+parser.add_argument('--api-key', type=str, default=None,
```

Comment:
(Someone could still pass an empty string as a value, but they’d really need to mean it)